src/main/java/io/github/reconsolidated/zpibackend/utils/BooleanListToStringConverter.java

```diff
@@ -0,0 +1,31 @@
+package io.github.reconsolidated.zpibackend.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Converter
+public class BooleanListToStringConverter implements AttributeConverter<List<Boolean>,String> {
+
+    private final static String SEPARATOR = ",";
+    @Override
+    public String convertToDatabaseColumn(List<Boolean> attribute) {
+        return attribute == null ? null : StringUtils.join(attribute,SEPARATOR);
+    }
+
+        @Override
```

Comment:
this indentation looks weird, did linter accept that?

Reply:
yes